src/Controller/HomeController.php

```diff
@@ -70,13 +75,36 @@ function (int $page = null) use ($request) {
 
     private function createFirstPage(array $arguments) : Response
     {
-        $arguments['carousel'] = $this->get('elife.api_sdk.covers')
+
+        $arguments['heroBanner'] = $this->get('elife.api_sdk.covers')
             ->getCurrent()
-            ->map($this->willConvertTo(CarouselItem::class))
-            ->then(Callback::emptyOr(function (Sequence $covers) {
-                return new Carousel($covers->toArray(), new ListHeading('Highlights', 'highlights'));
-            }))
-            ->otherwise($this->softFailure('Failed to load covers'));
+            ->slice(0, 1)
+            ->then(function (Sequence $items) {
+                /** @var Cover $item */
+                $item = $items[0];
+
+                /** @var ArticleVersion $article */
+                $article = $item->getItem();
```

Comment:
The cover item is not expected to be an ArticleVersion. It can currently be any of the following: https://github.com/elifesciences/api-raml/blob/develop/src/model/cover-list.v1.yaml#L23-L65

Reply:
Fixed